jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/WebAppContext.java

```diff
@@ -1486,38 +1465,31 @@ public MetaData getMetaData()
         return _metadata;
     }
 
-    public static void addServerClasses(Attributes attributes, String... pattern)
-    {
-        addClasses(__dftServerClasses, SERVER_SRV_CLASSES, attributes, pattern);
-    }
-
-    public static void addSystemClasses(Attributes attributes, String... pattern)
+    /**
+     * Add a Server Class pattern to use for all ee9 WebAppContexts.
+     * @param server The {@link Server} instance to add classes to
+     * @param patterns the patterns to use
+     * @see #getServerClassMatcher()
+     * @see #getServerClasses()
+     * @deprecated use {@link WebAppClassLoading#addProtectedClasses(Server, String...)}
+     */
+    @Deprecated(since = "12.0.8", forRemoval = true)
+    public static void addServerClasses(Server server, String... patterns)
```

Comment:
This is a signature change from Attributes to Server.
Also, the other methods that take or return ClassMatcher in this class now have signature changes too.
And the names are not consistent with the new hidden/protected in this method (eg:) setServerClasses still exists)